http-server/src/main/java/com/facebook/airlift/http/server/Authorizer.java

```diff
@@ -5,5 +5,5 @@
 
 public interface Authorizer
 {
-    AuthorizationResult authorize(Principal principal, Set<String> allowedRoles);
+    AuthorizationResult authorize(Principal principal, Set<String> allowedRoles, String requestUrl);
```

Comment:
Wondering which implementation of `Authorizer` will start using the `requestUrl` param. If there is a separate PR for the same can you please link it to this PR.

Reply:
It's in this PR: [facebookexternal/presto-facebook #1055](https://github.com/facebookexternal/presto-facebook/pull/1055)

We the incoming requests cannot pass the authorization check, instead of failing the requests, we just log the information and let them pass. (`Request Url` a key information we want to record)